### DISCUSSION TITLE
Global water vapor value also used by Sentinel-2?

### DISCUSSION THREAD

**Discussion by griembauer** [davidfrantz/force#311] [Q&A] [answered]
I am creating L2 ARD using the `force-level2` command from a mix of Landsat 8/9 and Sentinel-2 scenes (both are stored in the level1 folder). For now, I didn't bother to create a Water Vapor Database and only assign a global water vapor value in the parameter file, e.g.
```
# WATER VAPOR CORRECTION OPTIONS
# ------------------------------------------------------------------------
# This is the directory where the water vapor tables are located. Water
# vapor tables are not required for Sentinel-2, in this case DIR_WVPLUT
# may be NULL. For Landsat, it is recommended to use this functionality.
# As a minimum requirement, DIR_WVPLUT may be NULL and a global value
# for WATER_VAPOR needs to be specified. If a directory is given, 
# WATER_VAPOR is ignored. DIR_WVPLUT must contain water vapor tables.
# The 12 climatology tables must exist at least. They are used if the
# daily tables do not exist or if there is no valid daily value.
# Type: full directory path
DIR_WVPLUT = NULL
# This specifies a global value for atmospheric water vapor content in
# g cm-2. This parameter can be a dummy value to quickly process an image
# without needing to generate a water vapor database. Note that especially
# Landsat-8 is relatively insensitive to atmospheric water vapor (depending
# on wavelength), and external water vapor is not needed to process
# Sentinel-2. The error in using a dummy value is significant for the TM
# sensors.
# Type: Float. Valid range: [0,15]
WATER_VAPOR = 1.5
```
My question is: Is this `WATER_VAPOR` value applied to the Sentinel-2 atmospheric correction as well or is the standard Sentinel-2 watervapor channel used in this case (I suppose the latter would be preferable)? I can't really find the answer in the documentation

**Comment by ernstste** [comment]
Hi,

see here: https://force-eo.readthedocs.io/en/latest/howto/wvdb.html

> Background[](https://force-eo.readthedocs.io/en/latest/howto/wvdb.html#background)
> 
> During atmospheric correction, the effect of water vapor absorption can only be corrected if we know the amount of water vapor in the atmosphere.
> 
> If you are using Sentinel-2 data only, you can stop reading. Sentinel-2 is equipped with a water vapor channel, and thus, water vapor amount can be estimated from the images.

Stefan

**Comment by griembauer** [comment]
Hi Stefan,
Thanks for your reply! I am aware of the tutorial to create a water vapor database, but for simplicity I first wanted to go with the global water vapor value. 
My question is rather whether using a global `WATER_VAPOR` value in the parameter file overwrites the procedure of getting the Sentinel-2 water vapor from its own channel when using **both Sentinel-2 and Landsat-8/9** as input during Level2 processing. 

From https://force-eo.readthedocs.io/en/latest/howto/wvdb.html:
_If you are using Sentinel-2 data **only**, you can stop reading. Sentinel-2 is equipped with a water vapor channel, and thus, water vapor amount can be estimated from the images._
--> what if it is not **only** Sentinel-2?

Best,
Guido

**Comment by ernstste** [comment]
Hi Guido,

okay I see where you're coming from now.
As far as I recall S2 procesing will always use the WVP value estimated from the image. Don't take my word for it though, let's have @davidfrantz confirm.

Stefan

**Comment by davidfrantz** [comment, marked as answer]
Hi @griembauer,

yes, the global water vapor value won't be used for Sentinel-2.

But also for Landsat, I would suggest to download the existing water vapor database, which provides a better approximation than a single dummy value - even if not up-to-date.
https://zenodo.org/records/4468701#.YBA3LuhKgmI

Cheers,
David

**Reply by griembauer** [reply]
Great! Thanks a lot for the clarification!
